estate/models/estate_property.py

```diff
@@ -0,0 +1,128 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _
+from odoo import api
+from odoo import fields
+from odoo import models
+from odoo.exceptions import UserError
+from odoo.exceptions import ValidationError
+from odoo.tools import float_is_zero
```

Comment:
If you are importing things from same, need to write them together alphabetically..
```suggestion
from odoo import _, api, fields, models
from odoo.exceptions import UserError, ValidationError
from odoo.tools import float_is_zero
```